src/main/java/org/terasology/dynamicCities/construction/Construction.java

```diff
@@ -99,6 +100,7 @@
 import org.terasology.world.block.entity.placement.PlaceBlocks;
 import org.terasology.world.generation.Border3D;
 import org.terasology.world.generation.facets.SurfaceHeightFacet;
+import sun.misc.Request;
```

Comment:
Is this import intended? I general, you should avoid the sun packages if possible.